config/rbac/role.yaml

```diff
@@ -36,6 +43,14 @@ rules:
   - clusterversions
   verbs:
   - list
+- apiGroups:
```

Comment:
hm does it work, I was sure that for cluster scope resources you need cluster role

Reply:
let me check the annotations

Reply:
note `config/rbac/role.yaml` is deceptive: we actually have a `ClusterRole` in here